test/providers/summary/scancode.js

```diff
@@ -3,12 +3,12 @@
 
 const assert = require('assert')
 const summarizer = require('../../../providers/summary/scancode')()
-summarizer.logger = { info: () => {} }
+summarizer.logger = { info: () => { } }
 const fs = require('fs')
 const path = require('path')
 const { get, uniq, flatten } = require('lodash')
 
-const scancodeVersions = ['2.2.1', '2.9.2', '2.9.8', '3.0.0', '3.0.2']
+const scancodeVersions = ['2.2.1', '2.9.2', '2.9.8', '3.0.0', '3.0.2', '30.1.0']
```

Comment:
another nit, feel free to leave be; depending on your coding guidelines, I know I'd prefer to have all the array items here be their own unique lines, to make future diffs more clear what is changing

Reply:
I think that may make our linter mad, but I will give it a try